src/main/java/dumshenko/daniil/Main.java

```diff
@@ -1,18 +1,35 @@
 package dumshenko.daniil;
 
-import static dumshenko.daniil.Util.removeOccurrences;
-
 public class Main {
 
     public static void main(String[] args) {
-        int[] array = {1, 2, 3, 4, 2, 5, 2};
+//        int[] array = {1, 2, 3, 4, 2, 5, 2};
+//
+//        int numToRemove = 2;
+//
+//        int[] result = removeOccurrences(array, numToRemove);
+//
+//        for (int num : result) {
+//            System.out.print(num + " ");
+//        }
+
+        Shape circle = new Circle(FigureType.CIRCLE, 10);
+
+        circle.displayInfo();
+        System.out.print("My area: ");
+        System.out.println(circle.calculateArea());
```

Comment:
nit: Can be simplified to a one-liner using the String concatenation.
```suggestion
        System.out.println("My area: " + circle.calculateArea());
```